tests/ecommerce/tests_views.py

```diff
@@ -144,15 +149,28 @@ def prepare_cart(self):
             reverse('cart_add'), {'quantity': 1, 'product': product.id},
         )
 
-    def test_save_to_db(self):
-        url = reverse(Page.CUSTOM_PAGES_URL_NAME, kwargs={'page': 'order'})
-        email, phone = 'test@example.com', '+7 (222) 222 22 22'
-        self.prepare_cart()
+    def do_order(self, email='', phone='') -> None:
```

Comment:
usually a customer places an order but doesn't fulfill it, so `make/place_order` will be better